.github/workflows/ci.yml

```diff
@@ -45,4 +45,28 @@ jobs:
           echo "Test on installed library"
           make submodules
           make image
-          docker run --rm -v $(pwd):/code -t cb-mpc bash -c 'make build && make install && make benchmark-build && make demos && export CGO_ENABLED=1 && make test-go && make test-go-race'
+          docker run --rm -v "$(pwd)":/code -t cb-mpc bash -lc '
+            set -euo pipefail
+            cd /code
+            # Never re-touch submodules inside the container; avoids git safe.directory pain.
+            # If you insist on touching git here, you MUST first: git config --global --add safe.directory /code
+            if ! command -v go >/dev/null 2>&1; then
+              apt-get update
+              apt-get install -y --no-install-recommends ca-certificates curl git
+              GO_VERSION="${GO_VERSION:-1.22.7}"
```

Comment:
What is our default go version? I am seeing a couple of different versions in different parts of the code. From 1.22 to 1.24. Would be great to make them all to be the same.